pkgs/desktops/pantheon/desktop/wingpanel-indicators/bluetooth/default.nix

```diff
@@ -18,13 +18,13 @@
 
 stdenv.mkDerivation rec {
   pname = "wingpanel-indicator-bluetooth";
-  version = "2.1.5";
+  version = "unstable-2020-10-01";
 
   src = fetchFromGitHub {
     owner = "elementary";
     repo = pname;
-    rev = version;
-    sha256 = "sha256-txEjChutJCse/WjgQEfo+MSeeGWdpHUABGCsAqK6i3o=";
+    rev = "95c05e6c44404ff13e506fe487ffab177d422bcf";
+    sha256 = "sha256-txEjChutJCse/WjgQEfo+MSeeGWdpHUABGCsAqK7i3o=";
```

Comment:
The sha256 should change right?

Reply:
wait what 🤔

Reply:
ooops, looks like I used a wrong rev...

Reply:
_also, why do both git and GitHub show the hash as changed if it is, in fact, the same..._

Reply:
oh well, I just forgot to commit the change :/